gazelle/workspace.bzl

```diff
@@ -22,6 +22,16 @@ def bazel_skylib_gazelle_plugin_workspace():
     """Loads dependencies required to use skylib's gazelle plugin"""
     bazel_skylib_workspace()
 
+    maybe(
+        http_archive,
+        name = "rules_license",
+        sha256 = "4531deccb913639c30e5c7512a054d5d875698daeb75d8cf90f284375fe7c360",
+        urls = [
+            "https://mirror.bazel.build/github.com/bazelbuild/rules_license/releases/download/0.0.7/rules_license-0.0.7.tar.gz",
```

Comment:
Does this still exist? I'm not sure we are pushing to that any more as we get more bzlmod'y

Reply:
At least for 0.0.7, it works